config/routes.rb

```diff
@@ -11,6 +11,24 @@
     end
   end
 
+  resources :users, only: [] do
+    member do
+      get 'api_token'
+    end
+  end
+  # API routes
+  namespace :api do
+    namespace :v1 do
+      # post 'auth/login', to 'authenticate#login'
```

Comment:
- Please remove all comments from the codebase. Comments make our code look ugly.

Reply:
Okay done.